This patch was copied from Debian.  It is needed on armhf.

--- a/build/make/configure.sh	2013-12-17 18:18:46.904410454 +0100
+++ b/build/make/configure.sh	2013-12-17 18:19:58.720758736 +0100
@@ -799,7 +799,6 @@
 
         case ${tgt_cc} in
         gcc)
-            CROSS=${CROSS:-arm-none-linux-gnueabi-}
             link_with_cc=gcc
             setup_gnu_toolchain
             arch_int=${tgt_isa##armv}
@@ -822,10 +821,6 @@
                     check_add_cflags -mfpu=neon #-ftree-vectorize
                     check_add_asflags -mfpu=neon
                 fi
-
-                if [ -z "${tune_cpu}" ]; then
-                    tune_cpu=cortex-a8
-                fi
             else
                 check_add_cflags -march=${tgt_isa}
                 check_add_asflags -march=${tgt_isa}
